.github/actions/package/linux/action.yml

```diff
@@ -72,9 +72,12 @@ runs:
 
         appimagetool -s deploy "$INSTALL_APPIMAGE_DIR"/usr/share/applications/*.desktop
         cp ~/bin/AppImageUpdate.AppImage "$INSTALL_APPIMAGE_DIR"/usr/bin/
-        appimagetool "$INSTALL_APPIMAGE_DIR"
-
-        mv "Prism_Launcher-$VERSION-$APPIMAGE_ARCH.AppImage" "PrismLauncher-Linux-$VERSION-${{ inputs.build-type }}-$APPIMAGE_ARCH.AppImage"
+        # FIXME(@getchoo): Validate AppStream information when https://github.com/probonopd/go-appimage/pull/379 is merged
+        mkappimage \
+          --no-appstream \
+          --updateinformation "gh-releases-zsync|${{ github.repository_owner }}|${{ github.event.repository.name }}|latest|PrismLauncher-Linux-$APPIMAGE_ARCH.AppImage.zsync" \
```

Comment:
I assume the only reason your fork is working now is because we manually specify this, and previously - for whatever reason - your setup wasn't letting `go-appimage` autodetect it and create the zsync file as it should and previously did here